### PR TITLE
Updated the generic Kubeflow upgrade guide for v0.7

### PR DESCRIPTION
**Pull request by sarahmaddox**
Fixes https://github.com/kubeflow/website/issues/1258

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1260)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-1260--competent-brattain-de2d6d.netlify.com/docs/upgrading/upgrade/

**Comment by sarahmaddox** [comment]
/assign @richardsliu 

@richardsliu Please would you check that these updates show the recommended upgrade procedure for Kubeflow in v0.7.0.

**Comment by sarahmaddox** [comment]
This PR is ready for review. I'm adding a hold so that we can merge it when we're ready to update the docs to v0.7.0.

/hold

File: content/docs/upgrading/upgrade.md [hunk 1]
```diff
@@ -4,42 +4,44 @@ description = "Upgrading your deployment to a later version of Kubeflow"
...
 
-Until version 1.0 Kubeflow makes no promises of backwards compatibility or 
+Until version 1.0, Kubeflow makes no promises of backwards compatibility or 
 upgradeability. Nonetheless, here are some instructions for updating your 
 deployments.
 
 ## Upgrading Kubeflow
 
-Updating your deployment is a two step process:
+Follow these steps to upgrade your Kubeflow deployment:
```
Comment (jlewi) [inline content/docs/upgrading/upgrade.md]: @richardsliu @sarahmaddox KFupgrades won't be working with 0.7.0. Should we remove this section altogether or provide some sort of disclaimer?

Comment (richardsliu) [reply content/docs/upgrading/upgrade.md]: These instructions are not using the new upgrades logic. We can make the disclaimer above more explicit.

Also should we ask the users to delete their existing clusters first? I think directly applying the packages won't work.

Comment (sarahmaddox) [reply content/docs/upgrading/upgrade.md]: Thanks both. Updated. PTAL.

File: layouts/shortcodes/kf-latest-version.html [hunk 2]
```diff
@@ -1 +1 @@
-v0.6.2
\ No newline at end of file
+v0.7.0-rc.4
```
Comment (richardsliu) [inline layouts/shortcodes/kf-latest-version.html]: We have rc.5 now - should we update this now or later? We are quite likely to have another rc today or tomorrow.

Comment (sarahmaddox) [reply layouts/shortcodes/kf-latest-version.html]: Well spotted. The change to the kf-latest-version file is already merged, with the value for RC5. I'll sync this PR. Then we can update the `kf-latest-version` to RC6 etc as individual PRs. (I put it into all the deployment PRs too, for review convenience.)

**Comment by sarahmaddox** [comment]
@richardsliu PTAL

**Pull request merged**